Completed the basic functionality of the APP:

JoshuaThotsana opened the pull request:
-Created A model that Maps the API response
	-Created a Network service that handles API Requests
	-Created a Repository that uses a Network Service
	-Created A table View and A table View cell to display weather dates
	-Created a screen to show weather information based on the date selected.
	-Created a loading indicator that gets displayed with animation while the data is loading.

Commit:
Completed the basic functionality of the APP:
	-Created A model that Maps the API response
	-Created a Network service that handles API Requests
	-Created a Repository that uses a Network Service
	-Created A table View and A table View cell to display weather dates
	-Created a screen to show weather information based on the date selected.
	-Created a loading indicator that gets displayed with animation while the data is loading.

KevashanGovender commented on Mars-Mission/Service/Service.swift:
```diff
@@ -0,0 +1,12 @@
+
+import Foundation
+
+protocol Service {
```
This could have a more descriptive name

KevashanGovender commented on Mars-Mission/Repository/Repository.swift:
```diff
@@ -0,0 +1,12 @@
+
+import Foundation
+
+protocol Repository {
```
This could have a more descriptive name

Commit:
Added an Error handler message:
	-Added a date formatter functionality
	-Formatted the date on the landing screen
	-Formatted the date on the WeatherDetailView
	-Added showServiceError protocol
	-Call showServiceError function upon unsuccessful API call.
